webui/react/src/pages/ExperimentList.tsx

```diff
@@ -246,6 +273,10 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
   }, [fetchExperiments, fetchLabels]);
 
   const { stopPolling } = usePolling(fetchAll, { rerunOnNewFn: true });
+  const { stopPolling: stopHiddenSelectExpPolling } = usePolling(fetchHiddenSelectedExperiments, {
+    interval: 30_000, // 30s
+    rerunOnNewFn: true,
+  });
```

Comment:
made another polling because adding `fetchHiddenSelectedExperiments` into `fetchAll` causes infinite API calls.
and `interval` is 30s because i think 5sec is too often for selected experiments